test/fixtures/validators.js

```diff
@@ -5,6 +5,9 @@ export default {
   allow: {
     name: { type: 'string', allow: [ 'foo', 'bar' ] }
   },
+  allowNull: {
+    name: { type: 'string', allow: null }
```

Comment:
I would write this with 2 fields, both with `allow: null` then run one with a value and one with `null`